server/.envexample

```diff
@@ -1,6 +0,0 @@
-DB_HOST=
```

Comment:
Please untrack this file. We don't want to delete it, we want to keep it in the folder.